Dutch translation website

Nowa-Ammerlaan opened the pull request:
Haven't tested building it yet, because I'm having problems installing hugo (https://bugs.gentoo.org/714022). But it should just work.

Nowa-Ammerlaan committed:
Volledige Nederlandse vertaling website

Full Dutch translation website

marioortizmanero commented:
Thank you for the contribution!

A few small things that are missing because I added them recently (they were easy to spot, as they were the only words I understood at all haha):

* `config.toml`: `description`, `tags`
* `data/nl/hero.yaml`: `btnText` shouldn't be a question (I think).
* `data/nl/about.yml`: is "Fully automatic" exactly the same as in English?
* `data/nl/blog.yml`: `byText`, `atText`, `readMoreButton`
* `data/en/wiki.yml`:  `moreTitle`, `moreDescription`, `developerTitle`, `developerDescription`
* Can you add a PSA in the contact section text to only send messages in Spanish or English?

Thank you again :)

Nowa-Ammerlaan commented:
> Thank you for the contribution!
> 
> A few small things that are missing because I added them recently (they were easy to spot, as they were the only words I understood at all haha):
> 
>     * `config.toml`: `description`, `tags`
> 
>     * `data/nl/hero.yaml`: `btnText` shouldn't be a question (I think).

It works both ways in Dutch, but I'll remove the '?' to keep it inline with the English and Spanish versions.
 
>     * `data/nl/about.yml`: is "Fully automatic" exactly the same as in English?

My bad, fixed it
 
>     * `data/nl/blog.yml`: `byText`, `atText`, `readMoreButton`
> 
>     * `data/en/wiki.yml`:  `moreTitle`, `moreDescription`, `developerTitle`, `developerDescription`

Oops, I missed some of these because I was on the 'translations' branch, which is 5 commits behind master. I rebased from master and I think I got them all now.

>     * Can you add a PSA in the contact section text to only send messages in Spanish or English?
 
Added

marioortizmanero commented:
It's perfect now, thanks! Merging this.
